tests/FSharp.Compiler.ComponentTests/EmittedIL/StaticOptimizations/String_Enum.fs.il.bsl

```diff
@@ -130,53 +130,61 @@
     .method public static string  'string<SByteEnum>'(valuetype assembly/String/SByteEnum 'enum') cil managed
     {
       
-      .maxstack  3
-      .locals init (valuetype assembly/String/SByteEnum V_0)
+      .maxstack  8
       IL_0000:  ldarg.0
-      IL_0001:  stloc.0
-      IL_0002:  ldloca.s   V_0
-      IL_0004:  constrained. assembly/String/SByteEnum
-      IL_000a:  callvirt   instance string [netstandard]System.Object::ToString()
-      IL_000f:  ret
+      IL_0001:  box        assembly/String/SByteEnum
```

Comment:
@brianrourkeboll :

I used an action which plain runs ` dotnet test ./FSharp.Compiler.Service.sln --filter EmittedIL" -c Release` and it caused this.

I still hope it might be something strange in the setup (as in, not using the build script) and I will see a CI failure later.
Because if not, I do not like this - your optimization got effectively reverted and I do not see why.